frontend/src/views/MovieAddScreen.vue

```diff
@@ -121,7 +121,7 @@ export default {
         if (this.image) {
           const data = new FormData();
           data.append('image', this.image);
-          await axios.post(`/api/cinema/movies-${movie.id}-upload-image`, data, imageConfig);
+          await axios.post(`/api/cinema/movies-${movie.id}-upload-image/`, data, imageConfig);
```

Comment:
i dont think it would work. You should at least add url from VITE_API_URL variable in the beginning as in previous urls, also  id should be between "/" according to our API endpint